### PR TITLE
documentation: ZENKO-1757_add_Cosmos

### PR DESCRIPTION
**Pull request by wabernat**
This PR adds a section on Cosmos and clarifies NFS ingestion flows.

I also stuck MetalK8s under Kubernetes, because it makes better
sense that way.

You need it because it's awesome. 

Fixes: ZENKO-1757

### CODE REVIEW

**Comment by wabernat** [comment]
@bert-e approve
@bert-e reset

File: docs/docsource/Operation-Architecture/Architecture/Cosmos.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+======
+
+Cosmos provides translation from CloudServer to the NFS and SMB/CIFS
+file system protocols. It provides a REST API to CloudServer and
+translates CloudServer requests into native NFS or SMB protocol
+requests. Cosmos responds to CloudServer using the S3 protocol. The
+anticipated use case for Cosmos is to enable CloudServer to detect
+changes made to a non-object-storage repository with NFS or SMB/CIFS
+access and to back up these changes to a cloud storage location on a
+regular schedule.
```
Comment (ssalaues) [inline docs/docsource/Operation-Architecture/Architecture/Cosmos.rst]: Just some clarification:
```
Cosmos provides translation from CloudServer to the NFS and SMB/CIFS
file system protocols. It provides a REST API to CloudServer and
translates CloudServer requests into native file system requests.
The anticipated use case for Cosmos is to enable CloudServer to detect
changes made to a non-object-storage repository with NFS or SMB/CIFS
access and to back up these changes to a cloud storage location on a
regular schedule. Cosmos ingests these changes into CloudServer 
using the S3 protocol.
```

**Review by ssalaues — CHANGES_REQUESTED** [review]
Just some clarifications otherwise looks good

File: docs/docsource/Operation-Architecture/Architecture/Cosmos.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/Operation-Architecture/Architecture/Cosmos.rst]: I have a pedantic beef with Cosmos ingesting something into CloudServer (even if cloudserver called it up). Will this work? 

 "Cosmos serves these changes for CloudServer ingestion over the S3 protocol."

Comment (ssalaues) [reply docs/docsource/Operation-Architecture/Architecture/Cosmos.rst]: Yeah that's totally fine.

**Review by ssalaues — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]

**Comment by ssalaues** [comment]
@bert-e status

**Comment by wabernat** [comment]
@bert-e approve

**Comment by wabernat** [comment]
@bert-e reset

**Comment by wabernat** [comment]
@bert-e reset

**Pull request merged**